x-pack/plugins/lens/public/editor_frame_service/editor_frame/workspace_panel/message_list.scss

```diff
@@ -9,6 +9,7 @@
   max-height: $euiSize * 20;
   width: $euiSize * 16;
   overflow-wrap: break-word;
+  word-break: break-word;
```

Comment:
Forgive me, @stratoula. I made a mistake in [my comment](https://github.com/elastic/kibana/issues/151107#issuecomment-1431851877). I had meant to say to use `overflow-wrap: break-word;`, but it appears that style is already in use here (which means there's something else wrong).

In order for the existing `overflow-wrap` style to be honored, you'll need to apply a `min-width: 0;` style (or `min-inline-size: 0;` if you're feeling fancy) to each message's containing `EuiFlexItem`. See the screenshot example below.

For the sake of keeping related styles together, it probably would make sense to move the existing `overflow-wrap` style from `.lnsWorkspaceWarningList` to this `EuiFlexItem` as well. After that, you should be able to remove the `word-break` style above altogether.

![image](https://user-images.githubusercontent.com/3884767/219406178-d29c071c-444b-49c8-a6e7-58ad9df289f8.png)

Reply:
@MichaelMarcialis thanx! Done!